package.json

```diff
@@ -2,38 +2,38 @@
   "name": "pdf.js",
   "type": "module",
   "devDependencies": {
-    "@babel/core": "^7.25.2",
-    "@babel/preset-env": "^7.25.4",
-    "@babel/runtime": "^7.25.6",
+    "@babel/core": "^7.25.7",
+    "@babel/preset-env": "^7.25.7",
+    "@babel/runtime": "^7.25.7",
     "@fluent/bundle": "^0.18.0",
     "@fluent/dom": "^0.10.0",
     "@jazzer.js/core": "^2.1.0",
     "@metalsmith/layouts": "^2.7.0",
     "@metalsmith/markdown": "^1.10.0",
     "autoprefixer": "^10.4.20",
     "babel-loader": "^9.2.1",
-    "caniuse-lite": "^1.0.30001662",
+    "caniuse-lite": "^1.0.30001667",
     "canvas": "^2.11.2",
     "core-js": "^3.38.1",
     "eslint": "^8.57.0",
```

Comment:
Wait, should we not update this one a little bit since version https://github.com/eslint/eslint/releases/tag/v8.57.1 is available now?

Reply:
Good find! Done in the new commit.